lib/cretonne/src/ir/instructions.rs

```diff
@@ -709,15 +781,12 @@ mod tests {
         use ir::types::*;
 
         let vts = ValueTypeSet {
-            min_lanes: 0,
-            max_lanes: 8,
-            min_int: 3,
-            max_int: 7,
-            min_float: 0,
-            max_float: 0,
-            min_bool: 3,
-            max_bool: 7,
+            lanes: BitSet16::from_range(0,8),
+            ints: BitSet8::from_range(4,7),
+            floats: BitSet8::from_range(0,0),
+            bools: BitSet8::from_range(3,7),
```

Comment:
Note that the old `ValueTypeSet` intervals were half-open: `min <= x < max`. We should document what `BitSet::from_range()` does. Half-open or closed.